### PR TITLE
Allows for offset from UTC to be supplied as an option

### PR DESCRIPTION
**Pull request by JoshBroomberg**
Can convert all incoming times to UTC time based on supplied input offset

### CODE REVIEW

**Comment by JoshBroomberg** [comment]
@khanotations this is ready for review by you

File: Gemfile.lock [hunk 1]
```diff
@@ -1,7 +1,7 @@
 PATH
   remote: .
   specs:
-    time_splitter (1.1.0)
+    time_splitter (1.3.0)
```
Comment (khannotations) [inline Gemfile.lock:4]: Lol why not 2..?

Comment (JoshBroomberg) [inline Gemfile.lock:4]: When I got the file it was at 1.2.0 so I bumped by 1. I have no idea why this is like this now.

**Comment by JoshBroomberg** [comment]
Any other comments? @khanotations. Once this is merged, it is a one line fix in ixstudents

**Pull request closed without merging**